mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -201,6 +201,13 @@ func resourceBigtableInstanceCreate(d *schema.ResourceData, meta interface{}) er
 		return err
 	}
 
+	// Check that num_nodes and autoscaling_config can't be both set.
+	for _, cluster := range conf.Clusters {
+		if cluster.NumNodes > 0 && cluster.AutoscalingConfig != nil {
+			return fmt.Errorf("Creating instance: only one of num_nodes or autoscaling_config can be set.")
```

Comment:
Include the cluster ID in the error message so that we know which cluster has the problem?

What resourceBigtableInstanceUpdate? Don't we need to check there as well?

Make sure the test covers both create and update. Thanks!

Reply:
We can't check this for Update because NumNodes will always be filled with >0 value when this check runs.
Let me update the error message.

Reply:
But... but... you just said it: "NumNodes will always be filled with >0", then why don't we check that?

What is an "update" you are talking about? If you are referring to cluster update, that is not what I was referring to. I was referring to the update in this instance resource. For instance update, a new cluster could be added, hence it is a cluster create.

Regarding to difference between create and update, we can deal with it later.

Reply:
Sure, but I'm unsure how to distinguish that.. it seems to me instance update includes both cluster create and cluster update? I'm not sure if we can just check that in `resourceBigtableInstanceUpdate()`, tho I can give it a try later today.